Homework-conditions-Amount/index.js

```diff
@@ -0,0 +1,133 @@
+        description: " Short,pleasantly loose cut ,100% cotton",
+        price: "15€",
+        rating: 4,
+        shippingCost: {
+            cost: freeShipping
+        }
+    }
+
+];
+let amountArray = [0, 1, 2, 3];
+
+let showCart = [];
+
+function dressListInput() {
+    for (let i = 0; i < dressList.length; i++) {
+        const dressElementDescr = document.createElement("div")
+        dressElementDescr.innerHTML = `<li> <h5>${dressList[i].name}</h5>
+    <img  class ="dressImage" src="${dressList[i].imageUrl}" />
+    <p>${dressList[i].description}</p>
+    <p>Price:${dressList[i].price}</p>
+    ${createStars(dressList[i].rating)}
+   <p>Delivery Cost :${dressList[i].shippingCost.cost.deliveryCharge}</p>
+    
+    </li>`;
+        dressElementDescr.classList.add("dressDescription");
+        const myAmountList = document.getElementById("myAmount");
+        const newAmount = document.createElement("input");
+        newAmount.type = "number";
+        newAmount.classList.add("inputEle")
+        newAmount.placeholder = "Enter the required Amount";
+        myAmountList.appendChild(newAmount);
+        console.log(newAmount.value);
+        newAmount.addEventListener("change", function newValueSel() {
+            if (newAmount.value > amountArray[3]) {
+                document.getElementById("selectDress").innerHTML = "In Stock";
+            } else if (newAmount.value == amountArray[3] && newAmount.value > 1) {
+                document.getElementById("selectDress").innerHTML = "Almost Out of Stock";
+            } else if (newAmount.value == amountArray[1]) {
+                document.getElementById("selectDress").innerHTML = "Last Chance";
+            } else if (newAmount.value == amountArray[0]) {
+                document.getElementById("selectDress").innerHTML = "Out of Stock";
+            }
+        });
+
+        const totalListDisplayBtnEle = document.getElementById("totalListBtn");
+        const addToCartBtn = document.createElement('button');
+        addToCartBtn.addEventListener('click', function() {
+            showCart.push({
+                newDress: dressList[i].name,
+                newPrice: dressList[i].price
+            });
+        });
+        addToCartBtn.innerText = "Add to Cart";
+        addToCartBtn.classList.add("cartBtn")
+        addToCartBtn.classList.add("btn-toolbar")
+        dressElementDescr.appendChild(addToCartBtn);
+        dressListCmp.appendChild(dressElementDescr);
+    }
+}
+dressListInput();
+
+function createStars(starCount) {
+    const maxHearts = 5; // the max rating possible
+    const unFilledStars = maxHearts - starCount; //. the number of unchecked stars
+    let html = ''; // using let because we are going to add to this variable 
+    // loop through star count a create the html for check stars
+    for (let i = 0; i < starCount; i++) {
+        html += '<span class="fa fa-star checked"></span>';
+    }
+
+    for (let i = 0; i < unFilledStars; i++) {
+
```

Comment:
not needed empty line

Reply:
required Amount means....The amount of dress items that the user wants to purchase ,according to the no of dress items...the message is displayed

Reply:
Ok, got it! But this logic as it is right now doesn't bring to much sense to the user. Because if I enter 1 it would say to me - last chance, but if I enter 100 it would say in stock. So the logic is the other way around here :)
You can change it in the following way: create another property for your dress object `amount` and make it different for different dresses. And then when you would generate the layout for the dress, check amount and display the message. This would give user an overview how many items is left in the shop.